ta4j-core/src/test/java/org/ta4j/core/analysis/criteria/RewardRiskRatioCriterionTest.java

```diff
@@ -61,7 +56,8 @@ public void rewardRiskRatioCriterion() {
         double peak = (105d / 100) * (100d / 95);
         double low = (105d / 100) * (90d / 95) * (80d / 95);
 
-        assertNumEquals(totalProfit / ((peak - low) / peak), rrc.calculate(series, tradingRecord));
+        AnalysisCriterion ratioCriterion = getCriterion();
```

Comment:
Just a note - you could make the `ratioCriterion` an instance field and put the `getCriterion()` call in an `@Before` or in the constructor.  Not necessary, just a comment.

Reply:
Yeah, I think it's related to my comment above, what do you think about https://github.com/ta4j/ta4j/pull/226#discussion_r176988229 ?